composer.json

```diff
@@ -14,7 +14,9 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.0"
+        "php": "^7.0",
+        "ext-memcache": "*",
```

Comment:
This should be in a suggest block so people who do not use memcache are not required to install it.

Reply:
Moved it to the require-dev section + added info in suggest block.